### ISSUE TITLE
collaborate with the Go team

### ISSUE THREAD

**Issue by arnehormann** [VividCortex/go-database-sql-tutorial#10]
There's apparently an effort underway to improve the database/sql documentation and provide examples.
Issue: https://groups.google.com/forum/#!topic/golang-dev/bUTGb77jYFs
Page: https://code.google.com/p/go-wiki/wiki/SQLInterface

This is a very, very vague issue, so please close if it's not helpful.

**Comment by xaprb** [comment]
Thanks. I had thought about proposing this repo's documentation to be
worked into the official docs but never took the initiative. Too busy. I am
happy to do that if it seems beneficial. Should I propose that on the
golang-nuts mailing list or the golang-dev list? Not sure what to do.

**Comment by mattrco** [comment]
Hi Baron, I've been working on this. The ticket itself is here: https://code.google.com/p/go/issues/detail?id=5886

As you can see, any major changes have been postponed until after 1.2.

I submitted a CL which you can follow here: https://codereview.appspot.com/14087043

This is a good resource so it would be great to integrate it into the wiki page or at least link to it. For the actual package documentation I suggest we pick this discussion up again after 1.2 and figure out the scope of the work.

**Comment by xaprb** [comment]
Certainly. Why don't you ping me when it's time? We can leave this issue
open for anyone who's interested to see.

**Comment by mattrco** [comment]
Sounds good. Will do.

**Comment by xaprb** [comment]
I've updated this repo's documentation and moved it to a site hosted at http://go-database-sql.org/ for nicer browsing.
